### ISSUE TITLE
Kzones incompatible with plasma 5.26?

### ISSUE THREAD

**Issue by gabrielfern** [gerritdevriese/kzones#12]
After updating to KDE plasma 5.26, although the zones are there, and we can move windows to them, doing so leaves the window in a weird state.
Well, actually I noticed for only some zone positions/layouts we experience this problem.
What happens to the window is basically the whole resolution/text sharpness of the window gets much worse, and seems like the window can't tell its size was changed.

Take this screenshot as an example:
![Screenshot_20221012_102109](https://user-images.githubusercontent.com/25506386/195355127-74b16c29-48b1-4a25-9c3c-cf0b8c3ecdf4.png)

These are the layouts I'm using. And for these 7 zones, seems like only for zones 1,2,4 of layout number 2 the problem doesn't occur when moving windows to them.

```json
[
    {
        "name": "Layout 1",
        "padding": 0,
        "zones": [
            {
                "name": "1",
                "x": 0,
                "y": 0,
                "height": 100,
                "width": 70
            },
            {
                "name": "2",
                "x": 70,
                "y": 0,
                "height": 100,
                "width": 30
            },
            {
                "name": "3",
                "x": 15,
                "y": 10,
                "height": 80,
                "width": 70
            }
        ]
    },
    {
        "name": "Layout 2",
        "padding": 0,
        "zones": [
            {
                "name": "1",
                "x": 0,
                "y": 0,
                "height": 100,
                "width": 25
            },
            {
                "name": "2",
                "x": 25,
                "y": 0,
                "height": 100,
                "width": 50,
                "indicator": {
                    "offset": {
                        "y": -100
                    }
                }
            },
            {
                "name": "3",
                "x": 15,
                "y": 10,
                "height": 80,
                "width": 70,
                "indicator": {
                    "offset": {
                        "y": 100
                    }
                }
            },
            {
                "name": "4",
                "x": 75,
                "y": 0,
                "height": 100,
                "width": 25
            }
        ]
    }
]
```

**Comment by gabrielfern** [comment]
Do you know if this issue is already reported to KDE?

**Comment by gerritdevriese** [comment]
Looks like it: https://bugs.kde.org/show_bug.cgi?id=461029
But in the meantime I've found a workaround, it will be in the next release (v0.5)

**Comment by NicoStellwag** [comment]
> Looks like it: https://bugs.kde.org/show_bug.cgi?id=461029 But in the meantime I've found a workaround, it will be in the next release (v0.5)

@gerritdevriese Hi, first of all, thanks for your great work!
Do you already have a rough idea when v0.5 will be released?
Or is the test branch safe to use in the meantime?

Thanks in advance!

**Comment by gerritdevriese** [comment]
v0.5 has been released, but the minimum version is now 5.27
